### PR TITLE
Updated attenuation tests

### PR DESCRIPTION
**Pull request by RebeccaStankus**
https://highfidelity.atlassian.net/browse/HIFI-645

### CODE REVIEW

File: tests/testUtilities/testUtils.ts [hunk 1]
```diff
@@ -140,6 +140,7 @@ export interface ZoneData {
 
 export interface AttenuationData {
```
Comment (sabrina-shanman) [inline tests/testUtilities/testUtils.ts:141]: So, does the returned data usually include "id", or is it "zone-attenuation-id"?

File: tests/smoke/rest.smoke.test.ts [hunk 2]
```diff
@@ -656,24 +656,28 @@ describe('HiFi API REST Calls', () => {
...
                 "listener-zone-id": zone1Data.id,
                 "source-zone-id": zone3Data.id,
                 "za-offset": -5
             };
             attenuation4Data = {
                 "attenuation": 0.5,
+                "frequency-rolloff": 12,
                 "listener-zone-id": zone1Data.id,
                 "source-zone-id": zone4Data.id,
                 "za-offset": -5
```
Comment (sabrina-shanman) [inline tests/smoke/rest.smoke.test.ts:683]: We should try testing `null` as a value for one of the frequency rolloffs and one of the attenuations, now that that's a feature.

**Review by sabrina-shanman — CHANGES_REQUESTED** [review]

File: tests/smoke/rest.smoke.test.ts [hunk 3]
```diff
@@ -708,7 +712,7 @@ describe('HiFi API REST Calls', () => {
             expect(responseJSON).toEqual([attenuation3Data]);
```
Comment (sabrina-shanman) [inline tests/smoke/rest.smoke.test.ts:712]: Hmm, if this is working, then I think the new documentation is wrong. It currently says the ID key is "zone-attenuation-id". I suspect this is a regression that I introduced.

File: tests/smoke/rest.smoke.test.ts (hunk 2 above)
Comment (RebeccaStankus) [reply tests/smoke/rest.smoke.test.ts:683]: Actually, when I tried inputting null, the fetch request failed.

**Review by sabrina-shanman — APPROVED** [review]

**Pull request merged**